File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#include <stdio.h>


int main(void) {
    char count = 0, result = 0;
    short number = 0;
    scanf("%hhd", &count);
    result = count;

    for(char i = 0; i < count; i++) {
        scanf("%hd", &number);
        if(number == 1) result--;
        else {
            for(short j = 2; j < number; j++) {
                if(number % j == 0) {
                    result--;
                    break;
                }
            }
        }
    }

    printf("%hhd", result);

    return 0;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
count = int(input())
result = count

numbers = list(map(int, input().split()))

for number in numbers:
    if(number == 1): result -= 1
    else:
        for j in range(2, number):
            if(number % j == 0):
                result -= 1
                break

print(result)